distribution/ddf-common/src/main/resources/bin/setenv

```diff
@@ -44,7 +44,11 @@ LINUX="Linux"
 
 if [ "$OS" != "${OS%$LINUX*}" ]; then
    if [ -e /dev/urandom ]; then
-      EXTRA_JAVA_OPTS="-Djava.security.egd=file:/dev/./urandom"
+      if [ -z "$EXTRA_JAVA_OPTS" ]; then
+         EXTRA_JAVA_OPTS="-Djava.security.egd=file:/dev/./urandom"
+      else
+         EXTRA_JAVA_OPTS="$EXTRA_JAVA_OPTS -Djava.security.egd=file:/dev/./urandom"
```

Comment:
is the `if` necessary? why not always do this?

Reply:
I don't believe it is necessary, I can remove it.

Reply:
it was merged like this for 2.13.X - however making this fix will not change any functionality so I think we should be okay to make your suggested change and then merge it.

Reply:
You are correct, the if is not necessary. However, this change was already merged for 2.13.X and master, at this point because no functionality is being changed by your requested fix it's probably just worth leaving it like this for now.